### DISCUSSION TITLE
Welcome to strapi-plugin-comments Discussions!

### DISCUSSION THREAD

**Discussion by cyp3rius** [VirtusLab-Open-Source/strapi-plugin-comments#22] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mo9a7i** [comment]
Better than communicating everything through an issue, discuss here, add issue to be resolved there 👍🏽
